youtube_dl/extractor/adobepass.py

```diff
@@ -16,21 +16,6 @@
 
 
 MSO_INFO = {
-    'DTV': {
-        'name': 'DIRECTV',
-        'username_field': 'username',
-        'password_field': 'password',
```

Comment:
the default `username_field` and `password_field` has been set to `username` and `password` to prevent deuplication of the values for every MSO that depend on watchTVeverywhere, so it whould be better to keep the values it explicitly for DIRECTV as it's independant from watchTVeverywhere.